pupil-spa/src/app/services/pupil-prefs/pupil-prefs.service.spec.ts

```diff
@@ -9,6 +9,8 @@ import { AuditService } from '../audit/audit.service';
 import { QuestionService } from '../question/question.service';
 import { QuestionServiceMock } from '../question/question.service.mock';
 import { AccessArrangements } from '../../access-arrangements';
+import { first } from 'rxjs/operators'
```

Comment:
I don't see this operator being used in the diff?

Reply:
i will look into this and provide an update...

Reply:
removed.  Ng linter is not picking this up